### ISSUE TITLE
jacbi-xml library conflicts with use of Hibernate 3.6.10.Final

### ISSUE THREAD

**Issue by davewichers** [jcabi/jcabi-xml#162]
I suspect this issue is similar to #73. Simply adding the latest version of this library to my pom (without any code changes anywhere), causes this during startup:

```
SEVERE: Exception sending context initialized event to listener instance of class [org.owasp.benchmark.helpers.Startup]
java.lang.ExceptionInInitializerError
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:264)
        at org.owasp.benchmark.helpers.Startup.contextInitialized(Startup.java:34)
        at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4763)
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5232)
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:753)
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:727)
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:695)
        at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:1016)
        at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1903)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.hibernate.HibernateException: Unable to get the default Bean Validation factory
        at org.hibernate.cfg.beanvalidation.BeanValidationActivator.applyDDL(BeanValidationActivator.java:127)
        at org.hibernate.cfg.Configuration.applyBeanValidationConstraintsOnDDL(Configuration.java:1704)
        at org.hibernate.cfg.Configuration.applyConstraintsToDDL(Configuration.java:1654)
        at org.hibernate.cfg.Configuration.secondPassCompile(Configuration.java:1445)
        at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1856)
        at org.owasp.benchmark.helpers.HibernateUtil.<init>(HibernateUtil.java:57)
        at org.owasp.benchmark.helpers.DatabaseHelper.<clinit>(DatabaseHelper.java:39)
        ... 16 more
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.hibernate.cfg.beanvalidation.BeanValidationActivator.applyDDL(BeanValidationActivator.java:118)
        ... 22 more
Caused by: org.hibernate.HibernateException: Unable to build the default ValidatorFactory
        at org.hibernate.cfg.beanvalidation.TypeSafeActivator.getValidatorFactory(TypeSafeActivator.java:383)
        at org.hibernate.cfg.beanvalidation.TypeSafeActivator.applyDDL(TypeSafeActivator.java:109)
        ... 27 more
Caused by: javax.validation.ValidationException: Unable to create a Configuration, because no Bean Validation provider could be found. Add a provider like Hibernate Validator (RI) to your classpath.
        at javax.validation.Validation$GenericBootstrapImpl.configure(Validation.java:271)
        at javax.validation.Validation.buildDefaultValidatorFactory(Validation.java:110)
        at org.hibernate.cfg.beanvalidation.TypeSafeActivator.getValidatorFactory(TypeSafeActivator.java:380)
        ... 28 more
```

I'm using:
```
        <dependency>
            <groupId>org.hibernate</groupId>
            <artifactId>hibernate-core</artifactId>
            <version>3.6.10.Final</version>
        </dependency>
```

and same version of <artifactId>hibernate-entitymanager</artifactId> too.

No clue if this is enough info for you to figure out/fix the problem, but wanted to let you know about it.

**Comment by yegor256** [comment]
@davewichers try the latest version just released, it should work just fine I believe: 0.24.0

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-xml#162` is not assigned, can't get performer
